src/pages/Profile/index.js

```diff
@@ -23,90 +37,336 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
```

Comment:
In general, methods defined in the `React.Component` constructor need to be manually binded with `this` prepended to each method name. What if we moved these methods outside of the constructor to avoid manual binding? We follow a similar design within other page components.